patches/third_party-blink-public-mojom-worker-worker_content_settings_proxy.mojom.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom b/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom
+index b2629579ad0c02eafb44ff0d62f0ad4c74d96b38..bf3d36aeb2bdf939c83475e3877f448d8431823d 100644
+--- a/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom
++++ b/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom
+@@ -28,4 +28,10 @@ interface WorkerContentSettingsProxy {
+   // Returns whether the worker is allowed access to the file system.
+   [Sync]
+   RequestFileSystemAccessSync() => (bool result);
++
++  // Returns whether the worker is allowed access to privileged functions that could be used for fingerprinting.
++  [Sync]
++  AllowFingerprinting() => (bool result);
++  [Sync]
++  GetBraveFarblingLevel() => (uint8 result);
```

Comment:
please do this via chromium_src override
https://github.com/brave/brave-browser/wiki/Patching-Chromium#patching-mojom-files
an example of how to extend a class with new methods is here:
https://github.com/brave/brave-core/blob/569ccb3766a1e7752b7a4166bd3f07aad2afe560/chromium_src/brave/mojo/brave_ast_patcher/test_module.mojom#L89-L109

Reply:
fixed in https://github.com/brave/brave-core/pull/11452/commits/5e121e75da32f14a03267769012b31fc80021179

Reply:
It's a Fintech company
 
 Register with the link and visit "Neufund"  https://neufund.co/r/?a=jamesbethel66@gmail.com

It's actually a cryptocurrency investment company , they also operate on real estate, oil asset and mortgages as a back up financial aid